.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+
+.idea/
```

Comment:
What is this?

Reply:
Oh this is a folder that webstorm creates but no one else needs it.